src/fetch.ts

```diff
@@ -20,3 +20,23 @@ export async function gql(...params: Parameters<RpcGraphQL['query']>) {
         return value;
     });
 }
+
+export async function _fetch(url:string, body: any){
+    const res = await fetch(url, {
+        body: JSON.stringify({
+            ...body
+        }),
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        method: 'POST',
+    });
+
+    const text = await res.text();
+    return JSON.parse(text, (_, value) => {
+        if (value && typeof value === 'object' && '__bigint' in value) {
+            return BigInt(value['__bigint']);
+        }
+        return value;
+    });
+}
```

Comment:
Since this shares a lot of the same code as the `gql` function above it, maybe we create a function `post` and call into that from `gql`. What do you think?